packages/xo-web/src/xo-app/backup-ng/restore/index.js

```diff
@@ -268,6 +270,8 @@ export default class Restore extends Component {
             collection={this.state.backupDataByVm}
             columns={BACKUPS_COLUMNS}
           />
+          <h3>{_('restoreLegacy')}</h3>
+          <RestoreLegacy />
```

Comment:
This should totally disappear if there are no legacy backups, right? So maybe you can put the `h3` in `RestoreLegacy` and handle it there.